app/scripts/components/analysis/define/index.tsx

```diff
@@ -431,7 +431,7 @@ export default function Analysis() {
                     value={end ? dateToInputFormat(end) : ''}
                     onChange={onEndDateChange}
                     min={dateToInputFormat(start)}
-                    max='2022-12-31'
+                    max='2023-12-31'
```

Comment:
Should the year be dynamic with current - 1 ?

Reply:
This is a good catch. This value seemed to be hard-coded with an intention https://github.com/NASA-IMPACT/veda-ui/commit/a40244d3276809b665f535209ea67fcdd39785a4#diff-7e1c0b42a5124126043b8ee857dd483459ebc27bd003244d0793649c817c1bbcR421 but I am not sure why, I would wait until @j08lue can confirm if we can chang the date. I revereted this change.